lib/msf/core/exploit/cmdstager.rb

```diff
@@ -9,7 +9,7 @@ module Msf
 module Exploit::CmdStager
 
   include Msf::Exploit::EXE
-  include Msf::Exploit::CmdStager::Http
+  include Msf::Exploit::CmdStager::HTTP
```

Comment:
This is the only place this name is used. Other than #13401, which hasn't merged yet.